scripts/gptGenerateJsonFromIssue.py

```diff
@@ -107,8 +107,19 @@ def ask_gpt_for_initiative_json(self, issue_body: str) -> str:
         logging.warn("GPT did not infer function calling, best-effort answer parsing will be performed")
         return response_message["content"]
 
+def get_new_initiative_details() -> str:    
```

Comment:
The premise is that having the title will improve the results of GPT?

Reply:
yes, as discussed offline we consider the title to be the display name, which we haven't passed on to GPT until now